extensions/amp-selector/1.0/README.md

```diff
@@ -0,0 +1,296 @@
+</BentoSelector>
+```
+
+```css
+.custom-styles {
+  width: "100px";
+  height: "400px";
+}
+```
+
+#### Props for `<BentoSelector>`
+
+**disabled, form, multiple, name**
+
+The attributes above behave the same way as they do on a standard HTML [`<select>`](https://developer.mozilla.org/en/docs/Web/HTML/Element/select) element.
+
+**keyboardSelectMode**
+
+The `keyboardSelectMode` attribute dictates the keyboard navigation behavior for options inside `<BentoSelector>`.
+
+<ul>
+  <li>
+    `none` (default): The tab key changes focus between items in the `<BentoSelector>`. The user must press enter or space to change the selection. Arrow keys are disabled.
+  </li>
+  <li>
+    `focus`: Tab key gives focus to `<BentoSelector>`. The user navigates between items with the arrow keys. Must press space or enter to change the selection.
+  </li>
+  <li>
+    `select`: Tab key gives focus to `<BentoSelector>`. The selection changes as the user navigates options with arrow keys.
+  </li>
+</ul>
+
+#### Props for `<BentoSelectorOption>`
+
+**option**
+
+Indicates that the option is selectable. If a value is specified, the contents of the value is submitted with the form.
+
+**disabled, selected**
+
+The attributes above behave the same way as they do on a standard HTML [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) element.
+
+#### Interactivity and API usage
+
+Bento components are highly interactive through their API. The `BentoSelector` component API is accessible by passing a `ref`:
+
+```javascript
+import React, {createRef} from 'react';
+const ref = createRef();
+
+
+function App() {
+  return (
+    <BentoSelector ref={ref}>
+      <BentoSelectorOption option="1">Option 1</BentoSelectorOption>
+      <BentoSelectorOption option="2">Option 2</BentoSelectorOption>
+      <BentoSelectorOption option="3">Option 3</BentoSelectorOption>
+      <BentoSelectorOption option="4">Option 4</BentoSelectorOption>
+    </BentoSelector>
+  );
+}
+```
+
+#### Actions
+
+The `BentoSelector` API allows you to perform the following actions:
+
+**selectBy(delta: number)**
+Closes the selector.
+
+```js
+ref.selectBy(1); // Select next option in DOM sequence.
+ref.selectBy(-2); // Select the option that is two previous in DOM sequence.
+```
+
+**toggle(optionValue: string, opt_select: boolean|undefined)**
+Toggles the option with the given `optionValue` to be selected or deselected based on `opt_select`. If `opt_select` is not present, then the option will be selected if currently not selected, and deselected if currently selected.
+
+```js
+ref.toggle("1"); // Toggle the item with the attribute `option="1"`.
+ref.toggle("2", true); // Select the item with the attribute `option="2"`.
+```
+
+#### Events
+
+**select**
+
+This event is triggered when the user selects an option.
+Multi-selectors and single-selectors fire this when selecting or unselecting options.
+Tapping disabled options does not trigger the `select` event.
```

Comment:
Instead of emitting a select event, the component calls a given `onChange` prop with keys `value` - array of selected values, and `option` - the option whose selected state was changed.